Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Sliders/SliderAxis.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+
+namespace Microsoft.MixedReality.Toolkit.UI
+{
+    /// <summary>
+    /// Describes in which axis to orient the slider
+    /// </summary>
+    [Serializable]
+    public enum SliderAxis
```

Comment:
Ah, along the lines of https://github.com/microsoft/MixedRealityToolkit-Unity/pull/6924#discussion_r358936001, we should definitely try to standardize out axis enums. Don't think it's worth blocking this PR right now though